core/controllers/reader.py

```diff
@@ -129,7 +129,8 @@ class ExplorationPage(base.BaseHandler):
                     'type': 'int',
                     'validators': [{
                         'id': 'is_at_least',
-                        'min_value': 1 # Version must be greater than zero.
+                        # Version must be greater than zero.
```

Comment:
This and other similar changes have been done because we introduced a lint check which allows trailing (inline) comments only if it's an allowed pragma defined in this [list](https://github.com/oppia/oppia/blob/fe9ab48ef897c83ae3b3b593399bc3ebb4699fe5/scripts/linters/pylint_extensions.py#L49).